code/modules/client/preference_setup/loadout/loadout_uniform.dm

```diff
@@ -169,8 +173,23 @@
 	path = /obj/item/clothing/under/rank/head_of_security/navyblue
 	allowed_roles = list("Head of Security")
 
+/datum/gear/uniform/security_contractors
+	display_name = "security contractor uniform selection"
+	description = "A selection of security contractor related uniforms."
+	path = /obj/item/clothing/under/rank/security/necropolis
+	allowed_roles = list("Security Officer", "Warden", "Head of Security","Detective", "Forensic Technician", "Security Cadet")
+
+/datum/gear/uniform/security_contractors/New()
+	..()
+	var/uniform = list()
+	uniform["Necropolis security uniform"] = /obj/item/clothing/under/rank/security/necropolis
```

Comment:
here should be the full corporation name
Necropolis Industries

same with the two below as well